Libraries/LibWeb/Crypto/CryptoAlgorithms.cpp

```diff
@@ -4372,7 +4372,7 @@ WebIDL::ExceptionOr<GC::Ref<CryptoKey>> ECDSA::import_key(AlgorithmParams const&
             // 1. Perform any key import steps defined by other applicable specifications, passing format, keyData and obtaining key.
             // TODO: support 'applicable specifications'
 
-            // 2. If an error occured or there are no applicable specifications, throw a DataError.
+            // 2. If an error occurred or there are no applicable specifications, throw a DataError.
```

Comment:
These two crypto ones seem to be typos in the spec. If you see numbered comments like this, please check the spec (which should be linked at the top of the function) to see if this is a typo there. If so, don't fix it unless you also submit a fix to the spec (and then mention that in the commit message).

Reply:
Done 🍏

Reply:
You missed one of them.

Reply:
Fixed